projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -97,9 +79,9 @@ export function checkQuoteListContainsQuoteId() {
     checkQuoteListContainsQuoteId.name
   );
   cy.get('@quoteId').then((quoteId) => {
-    cy.get(quoteListSelector + rowSelector)
-      .contains(codeCellSelector, `${quoteId}`)
-      .should('be.visible');
+    cy.get(quoteListComponentSelector).within(() => {
+      cy.get('tr').contains('td.cx-code', `${quoteId}`).should('be.visible');
```

Comment:
do we need td.cx-code or is .cx-code already unique?
```suggestion
      cy.get('tr').contains('.cx-code', `${quoteId}`).should('be.visible');
```